pkg/network/protocols/events/consumer_test.go

```diff
@@ -189,10 +187,7 @@ func newEBPFProgram(c *config.Config) (*manager.Manager, error) {
 		},
 	}
 	options := manager.Options{
-		RLimit: &unix.Rlimit{
-			Cur: math.MaxUint64,
-			Max: math.MaxUint64,
-		},
+		RemoveRlimit: true,
```

Comment:
no need for : `MapEditors: make(map[string]*ebpf.Map),`?

Reply:
There was no need before, so I am preserving that behavior.